browser/src/slideshow/Transition2d.ts

```diff
@@ -139,6 +140,17 @@ class Transition2d extends TransitionBase {
 		gl.bufferData(gl.ARRAY_BUFFER, positions, gl.STATIC_DRAW);
 	}
 
+	private getUniformLocation(value: string) {
```

Comment:
This returns something but has no declared return type - it also doesn't return in an error state... Did this pass linting?

Reply:
Solved on : https://github.com/CollaboraOnline/online/pull/11743/commits/4c9995ab6f6c255efcdad4ca0f46d7a7e9074545